Components/Bullet/src/OgreBullet.cpp

```diff
@@ -376,7 +376,8 @@ btRigidBody* DynamicsWorld::addTerrainRigidBody(Terrain* terrain, int group, int
     // transfer ownership to node
     auto objWrapper = std::make_shared<TerrainRigidBody>(rb, mBtWorld, hfdata);
     node->getUserObjectBindings().setUserAny("BtCollisionObject", objWrapper);
-    rb->setCollisionFlags(rb->getCollisionFlags() | btCollisionObject::CF_STATIC_OBJECT);
+    rb->setCollisionFlags(rb->getCollisionFlags() | btCollisionObject::CF_STATIC_OBJECT |
```

Comment:
its better to add a flag to `addTerrainRigidBody(..., debugDraw=true)`, so it is still possible to enable this when needed.

Reply:
Ah, right, will do just that.

Reply:
Should be fixed now